docs/whats-new.md

```diff
@@ -1,16 +1,16 @@
-1. **Reusable executors** - [Executors](executors.md) define the environment and other settings for executing jobs. They allow you to reuse configuration of your execution environment across jobs.
-1. **Conditional steps in jobs** - The new `when` clause under steps allows you to run [conditionally run steps](conditional-steps.md) only when certain parameter values are true.
-1. **Orbs** - Orbs are packages of CircleCI configuration that can be shared across projects. Orbs allow you to make a single bundle of jobs, commands, and executors that can reference each other and can be imported into a CircleCI build configuration and invoked in their own namespace. Orbs are registered with CircleCI, with revisions expressed using the [semver](https://semver.org/) pattern.
+# What's new in 2.1 configuration?
+1. **Workflows now share the config version with the rest of the config** - You no longer need a separate version under the `workflows` stanza in a configuration. This is because the config version only needs to be specified once at the top of your configuration file.
+2. **Deprecation of some previous beta syntax** - For the very few people who used very early alpha versions of CircleCI 2.0, there were different names for certain built-in commands and parameters. There was an alternate syntax for steps - for example `type: run ...` instead of `run: ...`. These are deprecated, and some of these now display warnings while others are disallowed (including the legacy step syntax). (TODO: get the specific list.)
+3. **Reusable custom commands** - [Commands](commands.md) are parameterized lists of steps you declare that can be invoked as a step in a job. This will allow you to reduce repetitiveness in your code and encapsulate common lists of steps, including the ability to call them with different parameters.
+4. **Parameters in jobs** - Jobs can now declare [parameters](parameters.md) that can be passed in when invoking the job in a workflow. This allows you to reuse jobs or call them differently based on parameter inputs. You can also now run the same job multiple times with different parameter values for each.
+5. **Reusable executors** - [Executors](executors.md) define the environment and other settings for executing jobs. They allow you to reuse a configuration of your execution environment across jobs.
+6. **Conditional steps in jobs** - The new `when` clause under steps allows you to run [conditionally run steps](conditional-steps.md) only when certain parameter values are true.
+7. **Orbs** - Orbs are packages of CircleCI configuration that can be shared across projects. Orbs allow you to make a single bundle of jobs, commands, and executors that can reference each other and can be imported into a CircleCI build configuration and invoked in their own namespace. Orbs are registered with CircleCI, with revisions expressed using the [semver](https://semver.org/) pattern.
```

Comment:
@thomasr8 FYI, in markdown we were using the "1." numbering because it renders as an ordered list, but that way you don't have to worry about reordering manually if things move around. Not going to hold up merging over that, but FYI...